benchmark/src/main/java/org/corfudb/benchmark/Common.java

```diff
@@ -0,0 +1,37 @@
+package org.corfudb.benchmark;
+
+import org.apache.commons.math3.random.BitsStreamGenerator;
+import org.apache.commons.math3.random.MersenneTwister;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.SMRMap;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+
+import java.util.Map;
+
+@State(Scope.Benchmark)
+public class Common {
+    Map<Long, byte[]> map;
+    CorfuRuntime rt;
+    final BitsStreamGenerator RND = new MersenneTwister();
+
+    @Setup(Level.Trial)
+    public void setup() {
+        rt = new CorfuRuntime("localhost:9000").connect();
+        map = rt.getObjectsView()
+                .build()
+                .setStreamName(Integer.toString(RND.nextInt()))
+                .setType(SMRMap.class)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "SMRMap"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)